deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -257,30 +257,19 @@ async def create_new_job(
     try:
         user_id = str(uuid.UUID(token["userid"]))
         logger.info(f"Creating pulse3d job for upload {details.upload_id} with user ID: {user_id}")
-
-        params = [
-            "baseline_widths_to_use",
-            "prominence_factors",
-            "width_factors",
-            "twitch_widths",
-            "start_time",
-            "end_time",
-        ]
-        # TODO could make this if condition `details.version <= "0.25.0"` using the semver package
-        if details.version != "0.24.6":
-            # max_y param was added in 0.25.0
-            params.append("max_y")
-        if details.version not in (
-            "0.24.6",
-            "0.25.1",
-        ):
-            # TODO add unit tests
-            # peaks_valleys was added in 0.25.2
-            params.append("peaks_valleys")
-
-        details_dict = dict(details)
-        analysis_params = {param: details_dict[param] for param in params}
-
+        analysis_params = {
```

Comment:
I don't think merging the RC branch in went as expected, this is removing a bunch of code that Luci just added. You'll need to fix the merge